### ISSUE TITLE
Equivalent of `osc release` in core.py missing

### ISSUE THREAD

**Issue by SchoolGuy** [openSUSE/osc#1182]
**Is your feature request related to a problem? Please describe.**

Currently as a consumer of the `core.py` I am not able to trigger a release in the OBS due to the functionality only being implemented in the `commandline.py` directly.

**Describe the solution you'd like**

I can call `core.release()` with the same parameters as on the command line.

**Describe alternatives you've considered**

None

**Additional context**

None.

**Comment by dmach** [comment]
Hi @SchoolGuy, you're absolutely right.
I'm planning to refactor business logic away from `commandline.py`.
Are there more functions besides `release()` you want to use?

**Comment by SchoolGuy** [comment]
Yes the `osc api` command would be nice. More could be possible but I am not aware of it atm.

**Comment by SchoolGuy** [comment]
Found the next one: `osc cat` is only implemented in the `commandline.py`.
